raco/backends/sql/catalog.py

```diff
@@ -169,6 +172,9 @@ def _get_unary_sql(self, plan):
                 return sel
             return sel.group_by(*g)
 
+        elif isinstance(plan, algebra.Distinct):
```

Comment:
```sql
SELECT DISTINCT * 
FROM (SELECT...)
```

Is that what you expect? I have two tests in `optimizer_tests` for this on a single query. Feel free to propose more queries.